pkg/config/setup/config.go

```diff
@@ -526,6 +526,8 @@ func InitConfig(config pkgconfigmodel.Config) {
 	config.BindEnvAndSetDefault("dogstatsd_context_expiry_seconds", 20)
 	config.BindEnvAndSetDefault("dogstatsd_origin_detection", false) // Only supported for socket traffic
 	config.BindEnvAndSetDefault("dogstatsd_origin_detection_client", false)
+	// If enabled, all origin detection mechanisms will be unified to use the same logic.
```

Comment:
I think this part might be confusing. Perhaps the command can specify that it overrides the others and what is the equivalent with the legacy config.

Reply:
Done https://github.com/DataDog/datadog-agent/compare/530b8ec7614c239b787fbba4c242347330a6c2ee..fb455ecd8b906ea9acd8639073e031ef84b6f9e5